README.md

```diff
@@ -79,11 +80,21 @@ Then you can fetch changes at any time from this remote:
 git pull upstream develop
 ```
 
-When you have finished editing, commit and push to your fork:
+To keep commit messages consistent, contributors are advised to configure `git` with the supplied commit message template by running the following command in the repository's directory:
```

Comment:
```suggestion
To keep commit messages consistent, contributors are advised to configure `git` with the commit message template provided. This can be achieved by running the following command in the repository's directory:
```